### DISCUSSION TITLE
Don't remove deterministic Order please

### DISCUSSION THREAD

**Discussion by panawel** [mobile-dev-inc/Maestro#1688] [General]
It's really useful for me, and it's working great that way.
Don't remove it in future versions please!

executionOrder:
      flowOrder:

Doesn't work at all.

**Comment by DavidREntwistle** [comment]
I feel both the executionOrder & deterministicOrder have good use cases for each, but not to be used together.

- `executionOrder` for maintaining a sequential ordering flows.
- `deterministicOrder: true` for alphabetically ordering flows.

The only part I'm not satisfied with is `deterministicOrder: false`, I'm yet to have a valid use case for this as it appears to be a staggered list that remains consistent rather than a random list each time.

The `deterministicOrder: true` is also better for larger suites running against different include/exclude tags, as `executionOrder` currently throws exceptions when a flow in the `flowOrder` has been excluded with the tags.
